source/rst/geom_series.rst

```diff
@@ -856,24 +855,24 @@ i.e., the fraction of income that is consumed
     plt.title('Changing Consumption as a Fraction of Income')
     plt.ylabel('$y_t$')
     plt.xlabel('$t$')
-    for b in (b_0, b_1, b_2,b_3):
-        plt.plot(np.arange(0,T+1),calculate_y(i_0,b,g_0,T,y_init),label= r'$b=$'+f"{b:.2f}")
+    for b in (b_0, b_1, b_2, b_3):
+        plt.plot(np.arange(0, T+1), calculate_y(i_0, b, g_0, T, y_init), label= r'$b=$'+f"{b:.2f}")
```

Comment:
There shouldn't be a space following `label= ` here.

Also I think this can be simplified to `label= rf'$b= {b:.2f}$'`